lib/rspec/expectations/fail_with.rb

```diff
@@ -26,6 +26,10 @@ def fail_with(message, expected=nil, actual=nil)
 
         RSpec::Support.notify_failure(RSpec::Expectations::ExpectationNotMetError.new message)
       end
+
+      def call(object)
+        RSpec::Matchers::Composable.surface_descriptions_in(object)
+      end
```

Comment:
Or prehaps assign the lamba to a constant and use that? Then it's only allocated once right?